avocado/plugins/yaml_to_mux.py

```diff
@@ -261,9 +261,9 @@ def run(self, args):
             debug = getattr(args, "mux_debug", False)
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
-            except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+            except IOError:
+                msg = 'Cannot access the multiplex file(s) in this host.'
+                logging.getLogger("avocado.app").warning(msg)
```

Comment:
Removing the `_copy_files()` means that the multiplex will be also affected. If the multiplex file is not available in local host, it does not mean it is not available remotely. Since there is no guarantee remote plugin will engage before yaml_to_mux plugin, we cannot properly probe it's a remote execution at this point. The same suggestion you gave (to have a remote test resolver) applies here as definitive solution.